agent/php_api_internal.c

```diff
@@ -382,6 +383,57 @@ PHP_FUNCTION(newrelic_get_trace_json) {
   nr_set_destroy(&fas_metadata.active_segments);
 }
 
+PHP_FUNCTION(newrelic_get_error_json) {
```

Comment:
Is the only purpose for this in integration testing?

Reply:
Yes - it is so the test can verify the transaction id injected into the error trace JSON has the correct value.  Since this value changes from test to test I added a way to get the value used for a given test run.